tests/HasPermissionsTest.php

```diff
@@ -493,15 +493,14 @@ public function calling_givePermissionTo_before_saving_object_doesnt_interfere_w
 
         \DB::enableQueryLog();
         $user2->save();
-        $querys = \DB::getQueryLog();
         \DB::disableQueryLog();
 
         $this->assertTrue($user->fresh()->hasPermissionTo('edit-news'));
         $this->assertFalse($user->fresh()->hasPermissionTo('edit-articles'));
 
         $this->assertTrue($user2->fresh()->hasPermissionTo('edit-articles'));
         $this->assertFalse($user2->fresh()->hasPermissionTo('edit-news'));
-        $this->assertSame(4, count($querys)); //avoid unnecessary sync
+        $this->assertSame(4, count(\DB::getQueryLog())); //avoid unnecessary sync
```

Comment:
Please import all the things. It's probably better to rely on the long facade class name.

Reply:
Fixed